packages/aws-appsync-auth-link/src/signer/signer.ts

```diff
@@ -1,22 +1,29 @@
 /*
-Copyright 2017 - 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Copyright 2017 - 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the License. A copy of the License is located at
     http://aws.amazon.com/apache2.0/
 or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and limitations under the License.
 */
 global.Buffer = global.Buffer || require('buffer').Buffer; // Required for aws sigv4 signing
 
-var url = require('url'),
-    crypto = require('aws-sdk/global').util.crypto;
+var url = require('url');
 
-var encrypt = function (key, src, encoding = '') {
-    return crypto.lib.createHmac('sha256', key).update(src, 'utf8').digest(encoding);
+var { Sha256 } = require('@aws-crypto/sha256-universal')
```

Comment:
The code you referenced uses [@aws-crypto/sha256-js](https://github.com/aws/aws-sdk-js-crypto-helpers/tree/master/packages/sha256-js). Like [@aws-crypto-sha256-universal](https://github.com/aws/aws-sdk-js-crypto-helpers/tree/master/packages/sha256-universal), it is also not synchronous (please see the examples in the README's I've linked to). The universal package that I'm using here is meant to be `A consistent interface for SHA256 across browsers and NodeJs`. 

Either way, wouldn't this still be a breaking change because the consumer will need to now import `aws-sdk` manually, in the event that they were relying on our import? At first glance this seems like an incompatible API change to me, but open to your feedback!!